packages/ui-kit/src/components/PieChart/PieChart.tsx

```diff
@@ -0,0 +1,321 @@
+          legend: {
+            display: !isLegendHidden,
+            position: legendPosition,
+            labels: {
+              usePointStyle: true,
+              pointStyle: '*',
+              pointStyleWidth: 8,
+              boxHeight: 6
+            }
+          },
+          customChartLabelsPlugin
+        }
+
+        const datasets = isDoughnut ? { cutout: '75%' } : { cutout: '0' }
+
+        if (chartRef.current) {
+          const chart = chartRef.current
+
+          chart.data.labels = labels
+          Object.assign(chart.data.datasets[0], {
+            type: variant,
+            data: values,
+            backgroundColor: chartColorPlatte,
+            ...datasets
+          })
+
+          chart.options.plugins = {
+            ...chart.options.plugins,
+            ...customPlugins
+          }
+
+          chart.update()
+          return
+        }
+
+        let config: ChartConfiguration<'pie' | 'doughnut'> = {
+          ...chartConfig,
+          type: variant,
+          data: {
+            labels: labels,
+            datasets: [
+              {
+                data: values,
+                backgroundColor: chartColorPlatte,
+                borderWidth: 0,
+                offset: 4,
+                hoverOffset: 20,
+                ...datasets
+              }
+            ]
+          },
+          options: {
+            ...chartConfig.options,
+            indexAxis: 'y',
+            responsive: true,
+            maintainAspectRatio: false,
+            layout: {
+              padding: 4
+            },
+            plugins: {
+              ...chartConfig.options?.plugins,
+              ...customPlugins
+            },
+            scales: {
+              x: {
+                display: false
+              },
+              y: {
+                display: false
+              }
+            }
+          },
+          plugins: [customCanvasBackgroundColor, customChartLabelsPlugin]
+        }
+
+        if (chartConfigProps) {
+          config = chartConfigProps(config)
+        }
+
+        chartRef.current = new ChartJS(canvasRef.current, config) as ChartJS
+        canvasRef.current.style.borderRadius = '0px'
+      },
+      [variant, theme, card, chartProps, chartConfig, isDoughnut, chartConfigProps]
+    )
+
+    const destroyChart = () => {
+      if (chartRef.current) {
+        chartRef.current.destroy()
+        chartRef.current = null
+      }
+    }
+
+    // Calculate the other value and add it to the leaderboardData
+    const fetchedData = React.useMemo(() => {
+      if (!leaderboardData || !counterData) {
+        return
+      }
+
+      const leaderboardTotalValue = leaderboardData?.leaderboard?.rows.reduce((a, b) => a + Number(b[1]), 0) ?? 0
+      const counterValue = Number(counterData?.counter?.value ?? '0')
+
+      if (counterValue > leaderboardTotalValue) {
+        leaderboardData?.leaderboard?.rows.push(['Other', (counterValue - leaderboardTotalValue).toString()])
```

Comment:
I think we need a prop for configuring the "Other" label. "Other" is a good default, though. Perhaps `otherLabel`?